README.md

```diff
@@ -12,12 +12,15 @@ An API for collection management
 
 ### Configuration
 
-| Environment variable         | Default   | Description
-| ---------------------------- | --------- | -----------
-| BIND_ADDR                    | :26000    | The host and port to bind to
-| GRACEFUL_SHUTDOWN_TIMEOUT    | 5s        | The graceful shutdown timeout in seconds (`time.Duration` format)
-| HEALTHCHECK_INTERVAL         | 30s       | Time between self-healthchecks (`time.Duration` format)
-| HEALTHCHECK_CRITICAL_TIMEOUT | 90s       | Time to wait until an unhealthy dependent propagates its state to make this app unhealthy (`time.Duration` format)
+| Environment variable           | Default         | Description
+| ------------------------------ | --------------- | -----------
+| BIND_ADDR                      | :26000          | The host and port to bind to
+| GRACEFUL_SHUTDOWN_TIMEOUT      | 5s              | The graceful shutdown timeout in seconds (`time.Duration` format)
+| HEALTHCHECK_INTERVAL           | 30s             | Time between self-healthchecks (`time.Duration` format)
+| HEALTHCHECK_CRITICAL_TIMEOUT   | 90s             | Time to wait until an unhealthy dependent propagates its state to make this app unhealthy (`time.Duration` format)
+| MONGODB_BIND_ADDR              | localhost:27017 | The MongoDB bind address
+| MONGODB_COLLECTIONS_DATABASE   | collections     | The MongoDB collections database
+| MONGODB_COLLECTIONS_COLLECTION | collections     | The MongoDB collections collection
```

Comment:
😱 Totally appropriate name, just thankfully you're not writing this in Java or you might be using the Collections framework to iterate over documents in the collections collection.

Reply:
I know what you mean - it did feel weird putting it in 😅